src/backend/tests/unmocked/organization.test.ts

```diff
@@ -190,4 +190,42 @@ describe('Team Type Tests', () => {
       expect(images.exploreAsGuestImage).toBe('uploaded-image2.png');
     });
   });
+
+  describe('Set Logo', () => {
+    const file1 = { originalname: 'image1.png' } as Express.Multer.File;
+    const file2 = { originalname: 'image2.png' } as Express.Multer.File;
+    it('Fails if user is not an admin', async () => {
+      await expect(
+        OrganizationsService.setLogoImage(file1, await createTestUser(wonderwomanGuest, orgId), organization)
+      ).rejects.toThrow(new AccessDeniedAdminOnlyException('update logo'));
+    });
+
+    it('Succeeds and updates the logo', async () => {
+      const testBatman = await createTestUser(batmanAppAdmin, orgId);
+      (uploadFile as Mock).mockImplementation((file) => {
```

Comment:
Since this is mocked please move it to the mocked test file

Reply:
I don't see a mocked test file in the backend. Should I create one somewhere? Should I also move the other mocked function in this file?